tests/contenttypes_tests/test_fields.py

```diff
@@ -65,6 +66,17 @@ def test_clear_cached_generic_relation_when_deferred(self):
         answer.refresh_from_db()
         self.assertIsNot(answer.question, old_question_obj)
 
+    def test_form_field(self):
+        class AnswerForm(forms.ModelForm):
+            question = forms.CharField()
+
+            class Meta:
+                model = AnswerEditableGFK
+                fields = ["question"]
+
+        form = AnswerForm()
+        self.assertIn("question", form.fields)
```

Comment:
I think we should update the test in order to better capture the previous behavior which was generic foreign keys were excluded from forms unless explicitly allowed . This is actually not what editable is about which either always includes or always excludes (and errors when trying to add it in).

The test should probably be:
```suggestion
    def test_form_field(self):
        class AnswerForm(forms.ModelForm):
            class Meta:
                model = Answer
                fields = "__all__"

        form = AnswerForm()
        self.assertNotIn("question", form.fields)

        class AnswerWithGFKForm(forms.ModelForm):
            question = forms.CharField()

            class Meta:
                model = Answer
                fields = ["question"]

        form = AnswerWithGFKForm()
        self.assertIn("question", form.fields)
```